06_deploy_masters.sh

```diff
@@ -7,12 +7,20 @@ set -eux
 
 export OS_TOKEN=fake-token
 export OS_URL=http://ostest-api.test.metalkube.org:6385/
+
+# Clean previously env 
+for node in `openstack baremetal node list | grep master | awk '{print $2}'`; do  
```

Comment:
It might be better to use jq as previously mentioned, then we don't have any hard-coded dependencies on the name (it probably will be -master, but in theory folks could call those nodes whatever they want..

E.g use 

```
for node in $(jq -r .nodes[].name ocp/master_nodes.json); do
...
```

Reply:
Oh, i see what you mean... sorry, I thought you just wanted to avoid calling baremetal node list twice.